visual-tests/src/test-fixtures/series/stacked-bar.js

```diff
@@ -9,26 +9,48 @@
 
     d3.csv('stackedBarData.csv', function(error, data) {
 
-        var categories = data.slice(1).map(function(d) { return d.State; });
-        var yMax = d3.max(data, function(d) {
-            var total = 0;
-            for (var prop in d) {
-                if (prop !== 'State') {
-                    total += Number(d[prop]);
-                }
-            }
-            return total;
+        // Collect all series keys.
+        var seriesKeys = data.reduce(function(previous, current) {
```

Comment:
I would have expected the data array to objects with the same keys for each item, is it possible for the keys to be different? If not then I think most of this code is not required.